### PR TITLE
Added cart and wishlist page

### PR DESCRIPTION
**Pull request by sashaboi**
Cart and wishlist page added.

Preview : cart page https://deploy-preview-2--boring-kalam-a16074.netlify.app/cart.html

Preview : wishlist page https://deploy-preview-2--boring-kalam-a16074.netlify.app/wishlist.html

### CODE REVIEW

File: cart.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                </li>
+                <ul class="rightmenu">
+                    <li>
+                        <div class="btn-grey">
+                            Login
+                        </div>
+                    </li>
+                    <li class="badge">
+                        <i class="fa fa-heart" style="font-size:28px"></i>
+                        <div class="badge-number">
```
Comment (vikrantkode) [inline cart.html:34]: Avoid inline style.

File: cart.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+                            2
+                        </div>
+                    </li>
+                    <li class="badge">
+                        <i class="fa fa-shopping-cart" style="font-size:28px"></i>
+                        <div class="badge-number">
```
Comment (vikrantkode) [inline cart.html:40]: same here

**Review by vikrantkode — CHANGES_REQUESTED** [review]
suggested some change
in UI, border radius is used for buttons in cart and for login is different. (make them same - personal suggestion.)

**Comment by vikrantkode** [comment]
Overall looks good to me.

File: styles.css [hunk 3]
```diff
@@ -328,3 +328,53 @@ li { cursor: pointer; }
     padding-top: 20px;
 }
 
+.heart-badge{
+    background-color: rgba(0,0,0,0) !important;
```
Comment (raieegulhane) [inline styles.css:332]: Try not to use !important in code unless absolutely necessary.

**Review by raieegulhane — COMMENTED** [review]
Overall looks good. There are 2 buttons for cart in preview. One icon and another normal button. Have any one.

**Pull request merged**